test/cpu_fusion.cpp

```diff
@@ -4012,3 +4012,47 @@ NGRAPH_TEST(cpu_fusion, backwards_batchmatmultranspose_tensor2_tensor2)
 #endif
 
 #endif
+
```

Comment:
These tests need to be guarded by NGRAPH_JSON_DISABLE. They will also need to be moved to core_fusion.